### PR TITLE
update CI

### PR DESCRIPTION
**Pull request by RhinosF1**

### CODE REVIEW

**Comment by MacFan4000** [comment]
I was doing it in my PR

**Comment by RhinosF1** [comment]
so I can fix what fails

**Comment by RhinosF1** [comment]
@MacFan4000: can you update the tests submodule to dev?

File: .github/workflows/workflow.yml [hunk 1]
```diff
@@ -20,41 +24,45 @@ jobs:
...
         run: |
           python --version
+          sopel --version
           pip show sopel_plugins.channelmgnt
       - name: Run tests
         run: |
-          flake8 sopel_channelmgnt/channelmgnt --max-line-length 265
-          pytest tests
-      - name: Run pip freeze
+          flake8 . --max-line-length 265 --exclude=*/__init__.py
```
Comment (MacFan4000) [inline .github/workflows/workflow.yml]: Excluding __init__.py means we are excluding the plug-in code

**Review by MacFan4000 — CHANGES_REQUESTED** [review]

**Comment by MacFan4000** [comment]
> @MacFan4000: can you update the tests submodule to dev?

done

**Review by MacFan4000 — APPROVED** [review]

**Pull request merged**